cert/PowerShell_CACreationScript.ps1

```diff
@@ -0,0 +1,105 @@
+﻿#Get to OpenSSL directory
```

Comment:
Since this is a script, can you add the repo preamble (see CONTRIBUTING in the root dir)